assembl/static2/js/app/components/common/flatList.jsx

```diff
@@ -111,6 +123,17 @@ class FlatList extends React.Component<FlatListProps> {
     const entities = items.edges;
     return (
       <div className={className}>
+        {id && loadPreviousMessage ? (
+          <Button
+            className="button-dark button-submit"
+            style={{ marginBottom: '20px' }}
+            onClick={() => {
+              window.location.href = window.location.href.slice(0, window.location.href.indexOf('#'));
```

Comment:
Why not use the `browserhistory.push` here instead of forcing essentially a refresh?

Reply:
I fixed it, I know use react router, much cleaner.